spec/reports/cost_report_spec.rb

```diff
@@ -91,28 +85,43 @@ def add_cluster_htitem(*htitems)
 
   describe "#cost_per_volume" do
     it "calculates cost per volume" do
-      expect(cr.cost_per_volume).to eq(cr.target_cost / cr.num_volumes)
+      # 3 volumes, target cost $10
+      add_cluster_htitem(spm, mpm, ht_allow)
+      expect(cr.cost_per_volume).to be_within(0.01).of(3.33)
```

Comment:
I also wanted to be explicit here about the expected result, rather than relying on other methods to compute it for us. That makes the tests somewhat more brittle, but that brittleness is kind of endemic in this class anyway given the fixture data..